### DISCUSSION TITLE
As a UI designer, how to contribute to this project?

### DISCUSSION THREAD

**Discussion by greatcat19** [mastodon/mastodon#24274] [Q&A]
Hello! I'm a designer and want to help this project.
Do we have Figma UI kit of this? Who should I talk to to join?

**Comment by mjankowski** [comment]
Per the contribution guidelines - https://github.com/mastodon/.github/blob/main/CONTRIBUTING.md#project-specific-considerations - there is a separate internal team for design/UI/UX/etc type stuff, and external contributions are not desired along those lines.
